tests/test_visitors/test_ast/test_statements/test_statements_with_no_effect.py

```diff
@@ -193,6 +194,28 @@ def test_statement_with_no_effect(
     assert_errors(visitor, [StatementHasNoEffectViolation])
 
 
+@pytest.mark.parametrize('code', [
+    module_template,
+])
+@pytest.mark.parametrize('statement', [
+    'x += x + 2',
```

Comment:
We also need to test: 
- `-=`
- `*=`
- `**=`
- `/=`
- `^=`
- `%=`
- `>>=`
- `<<=`
- `&=`
- `|=`

We also need to test (on the right side):
- `x - 1`
- `x * x`
- `x / 2`
- `2 + x`
- `2 / x`
- `x % 2`
- `test(x)` (should be allowed)
- `x.attr` (should be allowed)
- `x.method()` (should be allowed)
- `test(x) + 2` (should be allowed)
- `x.attr + 2` (should be allowed)
- `x.method() + 2` (should be allowed)
- etc

Reply:
added more tests